src/bun.js/event_loop.zig

```diff
@@ -1882,6 +1884,70 @@ pub const MiniEventLoop = struct {
 
         this.loop.wakeup();
     }
+
+    pub fn stderr(this: *MiniEventLoop) *JSC.WebCore.Blob.Store {
+        return this.stderr_store orelse brk: {
+            const store = bun.default_allocator.create(JSC.WebCore.Blob.Store) catch unreachable;
+            var mode: bun.Mode = 0;
+            const fd = if (Environment.isWindows) bun.FDImpl.fromUV(2).encode() else bun.STDERR_FD;
+
+            switch (bun.sys.fstat(fd)) {
+                .result => |stat| {
+                    mode = @intCast(stat.mode);
+                },
+                .err => {},
+            }
+
+            store.* = JSC.WebCore.Blob.Store{
+                .ref_count = 2,
+                .allocator = bun.default_allocator,
+                .data = .{
+                    .file = JSC.WebCore.Blob.FileStore{
+                        .pathlike = .{
+                            .fd = fd,
+                        },
+                        .is_atty = bun.Output.stderr_descriptor_type == .terminal,
+                        .mode = mode,
+                    },
+                },
+            };
+
+            this.stderr_store = store;
+            break :brk store;
+        };
+    }
+
+    pub fn stdout(this: *MiniEventLoop) *JSC.WebCore.Blob.Store {
+        return this.stdout_store orelse brk: {
+            const store = bun.default_allocator.create(JSC.WebCore.Blob.Store) catch unreachable;
```

Comment:
Instead of unreachable I think we have been siding on
```suggestion
            const store = bun.default_allocator.create(JSC.WebCore.Blob.Store) catch bun.outOfMemory();
```